models/yang/annotations/sonic-bgp-clear-annot.yang

```diff
@@ -0,0 +1,16 @@
+module sonic-bgp-clear-annot {
+
+    yang-version "1.1";
+
+    namespace "http://openconfig.net/yang/annotation/sbgpclear-annot";
```

Comment:
Can u check the namespace , I think sonic YANG it has to Azure right?
"http://github.com/Azure/"